lib/oxidized/model/onefinity.rb

```diff
@@ -0,0 +1,19 @@
+class OneFinity < Oxidized::Model
+  using Refinements
+
+  # Fujitsu 1finity
+
+  prompt /(\r?[\w.@_()-]+[>]\s?)$/
+
+  cmd :all do |cfg|
+    cfg.each_line.to_a[1..-3].join
+  end
+
+  cmd 'show configuration | display set | nomore' do |cfg|
```

Comment:
Could you try `cmd 'show configuration | display set | nomore'`(without do `|cfg| ... end`). If you do not need to manipulate the output, there is no need to add a block here.

Reply:
Thank you, I saw this in other models and assumed it was necessary, it's been fixed